pkg/kine/drivers/generic/generic.go

```diff
@@ -508,7 +500,7 @@ func (d *Generic) Count(ctx context.Context, prefix, startKey string, revision i
 	return rev.Int64, id, err
 }
 
-func (d *Generic) Create(ctx context.Context, key string, value []byte, ttl int64) (rev int64, err error) {
+func (d *Generic) Create(ctx context.Context, key string, value []byte, ttl int64) (rev int64, succeeded bool, err error) {
```

Comment:
Do we need the correction of Create in our v1.2.0? It might be better to have it land in a separate PR.

Reply:
See #187 .